### PR TITLE
Update version and toc file

### PR DESCRIPTION
**Pull request by jhchoilee**
I updated the document 
1. add 8.0 
2. change toc file

Thanks

### CODE REVIEW

File: docs/platform/toc_all.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # What is Tizen?
 ## [Overview](/platform/what-is-tizen/overview.md)
-## Profiles
-### [Wearable](/platform/what-is-tizen/profiles/wearable.md)
+## Devices
 ### [TV](/platform/what-is-tizen/profiles/tv.md)
+### [Custum](/platform/what-is-tizen/profiles/tizen_custom.md)
```
Comment (safir-srbd) [inline docs/platform/toc_all.md:5]:
```suggestion
### [Custom](/platform/what-is-tizen/profiles/tizen_custom.md)
```

**Review by safir-srbd — COMMENTED** [review]
@jhchoilee I have found one change. Please implement it. Thank you.

**Comment by jhchoilee** [comment]
Thanks for review. I updated and commit again

**Review by safir-srbd — APPROVED** [review]
@jhchoilee The changes look good to me!

**Pull request merged**